README.md

```diff
@@ -54,55 +54,55 @@ end
 ```
 > Note that our controller extends from ApplicationController.
 
-### <a name='custom-validations'> User tracking and custom validations
+### <a name='custom-validations'> Entity tracking and custom validations
 
 #### Validations before giving out a token? Override `authenticate_entity`:
 
 ```ruby
-# authentication_controller.rb
+# application_controller.rb
 def authenticate_entity(params)
-  user ||= User.find_by(email: params[:email])
-  return nil unless user.present? && user.valid_password?(params[:password])
+  entity = Entity.find_by(some_unique_id: params[:some_unique_id])
+  return entity if entity.present? && entity.valid_password?(params[:password])
```

Comment:
I prefer an explicit `nil` here. And the `nil` should be the branching case, not the base.

```ruby
def authenticate_entity(params)
  entity = Entity.find_by(some_unique_id: params[:some_unique_id])
  return nil unless entity.present? && entity.valid_password?(params[:password])
  entity
end
```
If you prefer `nil` to be the base result, it should be explicit anyhow, it makes it clearer.

```ruby
def authenticate_entity(params)
  entity = Entity.find_by(some_unique_id: params[:some_unique_id])
  return entity if entity.present? && entity.valid_password?(params[:password])
  nil
end
```